### ISSUE TITLE
Authed Mine Blocked

### ISSUE THREAD

**Issue by sergeantexploiter** [easylist/easylist#706]
AuthedMine is authorized and allows the user to opt in before the miner is initialised. You guys cannot issue tools without doing your research work first. An entire workstation is uninstalling  the extension until there is an update to fix this issue. In the mean time, we going to settle with other coin blocking extensions and Avast.

AuthedMine is different from the old CoinHive api.

**Comment by FireServ** [comment]
Agree with you. It should not be blocked.

**Comment by Pocketart** [comment]
Temporary solution : download the miner javascript code from coinhive, obfuscate it, and it will still work despite blocker lists and anti viruses.

**Comment by Khrin** [comment]
Good idea.

And still no one of you has provided a real reason of why this service shouldn't be blocked and a valid example page.

**Comment by FireServ** [comment]
1 Reason. I own a game, not everyone can afford to buy ingame currency with real money. So they agree to the script and start a miner via web browser on their pc. They will be rewarded for every 100k Hashes they generated. So they can get the ingame currency without spending money. All with the users permission.

**Comment by Pocketart** [comment]
See a more detailed documentation here https://coinhive.com/documentation/authedmine

**Comment by loadedlen** [comment]
I own a charitable website that uses in-browser mining to raise money for humanitarian aid, disaster relief, child sponsorship, among other worthy causes. The authedmine/coinhive script is fully disclosed and users must opt-in. No complaints about it on my end, but now I have to inform them to whitelist my domain with adblockers so they can make the world a better place. Seems like an unnecessary hoop to jump through. Its hard enough to explain crypto currency to people without having to reverse their opinion on a script that's labelled as malware when it isn't.

**Comment by DatDraggy** [comment]
I don't think it's a good idea to block authedmine.
It clearly asks for the users consent and if a user is willing to contribute they wouldn't know that it is even there because it's being blocked.

**Comment by theinfinitecode** [comment]
I agree that authedmine should not be blocked. I can understand that the normal scripts are, since sadly some websites abuse it - but please, at least allow us to ask our visitors if they would like to help.

**Comment by gpaulius** [comment]
I also have Authedmine on my website and give users points for mining. I fully disclose everything and Authedmine does not start with their consent anyway. So blocking this great tool creates additional hassle as I need to explain how to disable blocking to my users and some of them are not very tech savvy and it is very inconvenient for them.

**Comment by ikealol** [comment]
Agree that authedmine should not be blocked.

**Comment by ICTman1076** [comment]
I agree that this shouldn't be blocked. My life essentially depends on advertising, and providing authedmine as an alternative is great. It also clearly asks for permission, of which the text cannot be changed by the website owner in order to mislead them to agree to something they don't want, and privacy is respected to the moon and back - https://coinhive.com/info/privacy - ("We do not track users. We do not use cookies. We do not use any third party tracking ... We keep your IP address on our servers in working memory (RAM) only. In case of a missbehaving client, the IP address will be kept in RAM for at most 48h. All other IP adresses are purged shortly after you disconnect from our servers. We do not store your IP address."). Please consider unblocking :)

**Comment by Leseratte10** [comment]
Another feature besides website support: AuthedMine provides a captcha-like interface. This means, a webpage owner can require a certain amount of hashes to be mined (with the users consent, of course) until another thing on the web page can be used / clicked / whatever. This is a cool solution to replace the annoying captchas - click button, wait a few seconds for the scripts to mine, then do whatever you wanted to do without having to enter characters. And because of the computing power required, you still can't write scripts doing that multiple times per second. 

Also, there probably are not many real-life applications yet - this new site is fairly new, and adding it to adblockers will only harm more new pages using this technique. 

Techiniques like this should be encouraged by adblocker developers - they provide a valid alternative to ads, they do not annoy the user, and they ask for consent before doing anything.

**Comment by Khrin** [comment]
Given that "we ask for permission beforehand" isn't a valid reason for unblocking, that no one have bothered to give a valid example, and linking here a privacy policy isn't one, I have to assume that there isn't a point of discussion here and another issue report will closed.
